app/authentication/resources.py

```diff
@@ -0,0 +1,40 @@
+from flask import session
+from flask_restful import Resource
+
+from app.users.models import User
+from app.lib.response_util import buildOkResponse, _returnUser
```

Comment:
This was my bad in the user's resource, but could you import the response_util module as whole instead of the helper functions directly. Like I said a while ago, 90% of the time we import whole modules. Resource and model classes tend to be the exception at least in Inkling code.